packages/block-library/src/image/style.scss

```diff
@@ -59,3 +59,10 @@
 		margin-right: auto;
 	}
 }
+
+// Variations
+.is-style-circle-crop img {
+	// We use an absolute pixel to prevent the oval shape that a value of 50% would give
+	// to rectangular images. A pill-shape is better than otherwise.
+	border-radius: 9999px;
```

Comment:
```suggestion
	border-radius: 9999px;
	
	
	// If a browser supports it, we will switch to using a circular SVG mask here.
	@supports (mask-image: none) or (-webkit-mask-image: none) {
		border-radius: none;
		mask-image: url('data:image/svg+xml;utf8,<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg"><circle cx="50" cy="50" r="50"/></svg>');
		-webkit-mask-image: url('data:image/svg+xml;utf8,<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg"><circle cx="50" cy="50" r="50"/></svg>');
		mask-mode: alpha;
		-webkit-mask-mode: alpha;
		mask-repeat: no-repeat;
		-webkit-mask-repeat: no-repeat;
		mask-size: contain;
		-webkit-mask-size: contain;
		mask-position: center;
		-webkit-mask-position: center;
	}
```

^ So this is essentially all we'd need to add to implement the idea proposed over in https://github.com/WordPress/gutenberg/issues/16130#issuecomment-509729977. The only issue is that this fails our linter, because I think we're not supposed to use both single and double quotes inside of a `url()` value. 😩 

We could convert this SVG to `base64` instead, but I know it's inadvisable to have long obscure strings in the code. OR, we could host the SVG somewhere else in Gutenberg, and call it here, but that seems weird too. Any ideas?

Reply:
> The only issue is that this fails our linter, because I think we're not supposed to use both single and double quotes inside of a url() value. 😩

There's a CSS comment you can write to disable the linter for the next line. It exists for this purpose exactly: to disable the linter for completely valid edgecases. I can't recall the exact comment to write, but if you combine it with a comment explaining why it's being disabled, I think that's totally fine. 

Btw, I don't know if we need the -webkit- properties — postCSS is supposed to add them automatically. It might not, if the property is exotic enough, but in general it should work.

Reply:
> There's a CSS comment you can write to disable the linter for the next line. It exists for this purpose exactly: to disable the linter for completely valid edgecases.

Btw, I don't know if we need the -webkit- properties — postCSS is supposed to add them automatically. It might not, if the property is exotic enough, but in general it should work.

Good call on both counts! Just pushed these changes in c6e56ca. 👍